Formula/libpinyin.rb

```diff
@@ -0,0 +1,61 @@
+class Libpinyin < Formula
+  desc "Library to deal with pinyin"
+  homepage "https://github.com/libpinyin/libpinyin"
+  url "https://github.com/libpinyin/libpinyin/archive/2.6.0.tar.gz"
+  sha256 "2b52f617a99567a8ace478ee82ccc62d1761e3d1db2f1e05ba05b416708c35d2"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gnome-common" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "wget" => :build
```

Comment:
This looks very wrong, what is being downloaded during installation?

Reply:
Thanks for the review!

In fact, there are language models and dictionaries being downloaded. They are used to generate data files.

Reply:
That is not acceptable for a homebrew-core formula. They would need to be incorporated into the source release, or downloaded as a `resource`

Reply:
Ok. I'll investigate the resource stuff. Thanks for the suggestion!

Reply:
Done, I just force-pushed this branch to keep only one commit.